config/i18n.json

```diff
@@ -187,6 +187,7 @@
   },
   "Filter": {
     "Adept": "\\(Adept\\)",
+    "AdeptDescription": "Shows all Adept, Harrowed, and Timelost weapons.",
```

Comment:
@nev-r thoughts?

Reply:
It's a bit awkward that this mentions the words Harrowed and Timelost as if they were different from Adept weapons -- they *are* adept weapons, they just go by a slightly differently flavored name, and that this is an exhaustive list that may have to be updated with S22's raid. Maybe something like

> "Shows Adept weapons. Includes Timelost/Harrowed/... weapons from remastered raids".